Print the number of tokens generated

helunwencser opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #6771

This is useful for verifying the correctness of AttentionSink.

Differential Revision: [D65784095](https://our.internmc.facebook.com/intern/diff/D65784095/)

helunwencser committed:
Print the number of tokens generated

This is useful for verifying the correctness of AttentionSink.

Differential Revision: [D65784095](https://our.internmc.facebook.com/intern/diff/D65784095/)

[ghstack-poisoned]

davidlin54 commented on examples/models/llama/runner/eager.py:
```diff
@@ -91,10 +91,11 @@ def main() -> None:
         else runner.text_completion(
             prompt=args.prompt,
             temperature=args.temperature,
+            echo=True,
```
out of curiousity, what does `echo=True` do?

helunwencser replied:
When it is true, the returned token list will include prompt tokens as well.

helunwencser committed:
Update on "Print the number of tokens generated"


This is useful for verifying the correctness of AttentionSink.

Differential Revision: [D65784095](https://our.internmc.facebook.com/intern/diff/D65784095/)

[ghstack-poisoned]
